lib/protobuf/extension.ex

```diff
@@ -114,18 +114,12 @@ defmodule Protobuf.Extension do
 
   @doc false
   def __cal_extensions__(mods) do
-    mods
-    |> Enum.filter(fn mod ->
-      if to_string(mod) =~ ~r/\.PbExtension$/ && Code.ensure_loaded?(mod) do
-        function_exported?(mod, :__protobuf_info__, 1)
-      end
-    end)
-    |> Enum.map(fn mod ->
-      {mod, mod.__protobuf_info__(:extension_props)}
-    end)
-    |> Enum.reject(fn {_mod, props} -> is_nil(props) end)
-    |> Enum.each(fn {mod, props} ->
-      Enum.each(props.extensions, fn {_, ext} ->
+    for mod <- mods,
+        to_string(mod) =~ ~r/\.PbExtension$/,
```

Comment:
Two ideas, not blocking:

```suggestion
        mod |> Module.split() |> List.last() == "PbExtension"
```

or

```elixir
String.ends_with?(inspect(mod), ".PbExtension")
```

I think both solutions are a bit easier to read and more solid (that is, I am always always always scared of regexes) 😄

Reply:
I'll merge this PR, maybe we can do this in a future refactor :)